src/utils/mtev_confstr.c

```diff
@@ -169,6 +169,12 @@ mtev_confstr_parse_duration(const char *input, uint64_t *output,
     if(unit_str == input)
       return MTEV_CONFSTR_PARSE_ERR_FORMAT;
 
+    /* remove space(s) between number and unit string */
+    while(*unit_str && !isalpha(*unit_str)){
```

Comment:
There's no need to be using `isalpha` here. We don't care if a character is a letter or not; all we care about is if it is a space.

There's already a function in C, `isspace`, that will check exactly that. I'd recommend using that.

Reply:
switch to `isspace` function